{{cookiecutter.distribution_name}}/{{cookiecutter.package_name}}/_dummy_visualizer.py

```diff
@@ -2,48 +2,30 @@
 # ready to develop your plugin.
 import os
 import os.path
+import pkg_resources
 
 import pandas as pd
+import q2templates
+
+
+TEMPLATES = pkg_resources.resource_filename(
+    "{{cookiecutter.package_name}}", 'assets')
 
 
 def mapping_viz(output_dir: str, mapping1: dict, mapping2: dict,
                 key_label: str, value_label: str) -> None:
     df1 = _dict_to_dataframe(mapping1, key_label, value_label)
     df2 = _dict_to_dataframe(mapping2, key_label, value_label)
 
-    with open(os.path.join(output_dir, 'index.html'), 'w') as fh:
-        fh.write('<html><head>')
-        fh.write('<link rel="stylesheet" type="text/css" '
-                 'href="css/style.css" />')
-        fh.write('</head>\n')
-        fh.write('<body>\n')
-        fh.write('<h3>mapping1:</h3>\n')
-        fh.write(df1.to_html(index=False, classes='dummy-class'))
-        fh.write('<h3>mapping2:</h3>\n')
-        fh.write(df2.to_html(index=False, classes='dummy-class'))
-        fh.write('</body></html>')
+    index = os.path.join(TEMPLATES, 'index.html')
+    context = {
+        "mapping1": df1.to_html(index=False, classes='dummy-class'),
```

Comment:
Most/all of the other visualizations with a table included have something along the following lines, in order to format the HTML table correctly:

```python
table = d1.to_html(index=False, classes="table table-striped table-hover")
table = table.replace('border="1"', 'border="0"')
```

Can you add that for both of the tables in this viz?